src/modules/helpchan.ts

```diff
@@ -70,6 +72,18 @@ This help channel has been marked as **dormant**, and has been moved into the **
 If your question wasn't answered yet, you can claim a new help channel from the **Help: Available** category by simply asking your question again. Consider rephrasing the question to maximize your chance of getting a good answer. If you're not sure how, have a look through [StackOverflow's guide on asking a good question](https://stackoverflow.com/help/how-to-ask)
 `;
 
+const helpMessage = (channels: Channel[]) => `
+Help channels work a little differently on this server. So that people aren't talking over each other, only one person may ask for help in a channel at a time.
+• Send a message to a help channel in the ":white_check_mark: | Available Help Channels" category (**currently ${channels.join(
+	' and ',
+)}**).
+• Our bot will move your channel to ":hourglass: | Occupied Help Channels".
+• Someone will (hopefully!) come along and help you.
+• When your question(s) are resolved, type \`!close\`.
+
+Help channels will be automatically closed after 18 hours of inactivity. When a channel is closed, it moves into the ":zzz: | Dormant Help Channels" category to eventually be recycled back into the ":white_check_mark: | Available Help Channels" category.
```

Comment:
Could the `18 hours` be changed to automatically reflect the environment variable?